### ISSUE TITLE
How to achieve the best score on Cora Dataset?

### ISSUE THREAD

**Issue by John-Yao** [benedekrozemberczki/GraphWaveletNeuralNetwork#3]
I run the code with default parameters and got 74% on Cora dataset. Is there any difference with tensorflow -version-repo?

**Comment by benedekrozemberczki** [comment]
They did not use Chebyshev polynomial approximations in the actual experiments as they described. I did. That seems to be the difference.

**Comment by John-Yao** [comment]
Thanks!
Have you try pygcn ? I follow the author change the way of normalising adajency matrix, and got better result.(https://github.com/tkipf/pygcn/issues/20)
So do you think the paper valuable?

**Comment by zhenhuascut** [comment]
Thanks for the nice work. But I am also curious about it, the highest acc is 0.77 in this implementation. The highest acc in originally released implementation by TensorFlow is 0.82. The problem in this implementation is the acc will sharply drop down to 0.22 when training more time. But the TensorFlow version doesn't, why this happen, can you explain it? Thanks.

**Comment by githubxiaowei** [comment]
```
+---------------------+-----------------------------+
|      Parameter      |            Value            |
+=====================+=============================+
| Approximation order | 20                          |
+---------------------+-----------------------------+
| Dropout             | 0.500                       |
+---------------------+-----------------------------+
| Edge path           | ../input/cora_edges.csv     |
+---------------------+-----------------------------+
| Epochs              | 300                         |
+---------------------+-----------------------------+
| Features path       | ../input/cora_features.json |
+---------------------+-----------------------------+
| Filters             | 16                          |
+---------------------+-----------------------------+
| Learning rate       | 0.001                       |
+---------------------+-----------------------------+
| Log path            | ../logs/cora_logs.json      |
+---------------------+-----------------------------+
| Scale               | 1                           |
+---------------------+-----------------------------+
| Seed                | 42                          |
+---------------------+-----------------------------+
| Target path         | ../input/cora_target.csv    |
+---------------------+-----------------------------+
| Test size           | 0.200                       |
+---------------------+-----------------------------+
| Tolerance           | 0.000                       |
+---------------------+-----------------------------+
| Weight decay        | 0.001                       |
+---------------------+-----------------------------+

Wavelet calculation and sparsification started.


Normalizing the sparsified wavelets.

Density of wavelets: 0.04%.
Density of inverse wavelets: 0.2%.

Loss:   0%|          | 0/300 [00:00<?, ?it/s]Training.

GWNN (Loss=0.7932): 100%|██████████| 300/300 [00:07<00:00, 42.41it/s]

Scoring.

Test Accuracy: 0.7227
```

**Comment by Gray-3** [comment]
> Thanks for the nice work. But I am also curious about it, the highest acc is 0.77 in this implementation. The highest acc in originally released implementation by TensorFlow is 0.82. The problem in this implementation is the acc will sharply drop down to 0.22 when training more time. But the TensorFlow version doesn't, why this happen, can you explain it? Thanks.
Hi！Have you solved this problem？

**Comment by Serendipity-LC** [comment]
> Thanks for the nice work. But I am also curious about it, the highest acc is 0.77 in this implementation. The highest acc in originally released implementation by TensorFlow is 0.82. The problem in this implementation is the acc will sharply drop down to 0.22 when training more time. But the TensorFlow version doesn't, why this happen, can you explain it? Thanks.

Hi！I also encountered this problem. Have you solved it？

**Comment by zxc1230321** [comment]
Hi! I have the same problem. Have you solved it？[](url)

> > Thanks for the nice work. But I am also curious about it, the highest acc is 0.77 in this implementation. The highest acc in originally released implementation by TensorFlow is 0.82. The problem in this implementation is the acc will sharply drop down to 0.22 when training more time. But the TensorFlow version doesn't, why this happen, can you explain it? Thanks.

> Hi！I also encountered this problem. Have you solved it？

**Comment by Xusr1123** [comment]
> > Thanks for the nice work. But I am also curious about it, the highest acc is 0.77 in this implementation. The highest acc in originally released implementation by TensorFlow is 0.82. The problem in this implementation is the acc will sharply drop down to 0.22 when training more time. But the TensorFlow version doesn't, why this happen, can you explain it? Thanks.
> 
> Hi！I also encountered this problem. Have you solved it？

I have the same problem.
